### PR TITLE
migrate to vtm

### PR DESCRIPTION
**Pull request by pstorch**
TODOs

- [x] update arrow and map position while recording
- [x] correct position of waypoint marker (pushpin)
- [x] map rotation
- [x] arrow rotation
- [x] make MapsActivity smaller :wink: 
- [x] fix tests
- [ ] ~~share image of map does not render at all~~

### CODE REVIEW

File: src/main/java/de/storchp/opentracks/osmplugin/Startup.java [hunk 1]
```diff
@@ -38,6 +37,7 @@ public void onCreate() {
         //In debug builds: show thread and VM warnings.
         if (BuildConfig.DEBUG) {
             Log.d(TAG, "Enabling strict mode");
+            /*
```
Comment (dennisguse) [inline src/main/java/de/storchp/opentracks/osmplugin/Startup.java:40]: Remove? :)

File: src/main/java/de/storchp/opentracks/osmplugin/utils/MapUtils.java [hunk 2]
```diff
@@ -183,18 +198,8 @@ public static float deltaAngle(float angle1, float angle2) {
...
-        paint.setStrokeWidth(strokeWidth);
-        paint.setStyle(Style.STROKE);
-        return paint;
-    }
-
-    public static Polyline createPolyline(MapView mapView, int trackColor, int strokeWidth) {
-        return new Polyline(MapUtils.createPaint(trackColor,
-                (int) (strokeWidth * mapView.getModel().displayModel.getScaleFactor())
-        ), AndroidGraphicFactory.INSTANCE);
+    public static PathLayer createPolyline(Map map, int trackColor, int strokeWidth) {
```
Comment (dennisguse) [inline src/main/java/de/storchp/opentracks/osmplugin/utils/MapUtils.java:201]: Inline?

**Review by dennisguse — APPROVED** [review]
Awesome! 🍾🚨

**Comment by pstorch** [comment]
Merged for broader testing in the nightlies

**Pull request merged**